src/main/java/org/openjdk/jextract/impl/StructLayoutComputer.java

```diff
@@ -86,6 +86,10 @@ void startBitfield() {
     void processField(Cursor c) {
         boolean isBitfield = c.isBitField();
         long expectedOffset = offsetOf(parent, c);
+        if (offset > expectedOffset) {
+            // out-of-order field, skip
+            return;
```

Comment:
What happens here is that, typically, we already have generated a padding _before_ (e.g. because we jumped into a struct, and saw that first offset is e.g. 42). So we later go back and see offset 0, so we try to emit something for which we already have emitted padding. The layout will be usable, but some fields will be skipped.